content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-2.0.md

```diff
@@ -49,3 +101,18 @@ table th:nth-of-type(2) {
+| **Network Operation**                        | **Member Registration Approval** - A network operator can now configure membership groups so that the operator is required to manually approve (or decline) member registration requests. For more information, see [Member Registration Approval]({{< relref "../operating/registration-approval.md" >}}). <br> <br> **Operational Statuses** - Four new operational statuses have been added for virtual nodes to provide fine-grained control over their flows and what components they can interact with: <ul><li>`flowOperationalStatus` — describes a virtual node's ability to start new flows. </li> <li>`flowP2pOperationalStatus` — describes a virtual node's ability to communicate with peers.</li> <li>`flowStartOperationalStatus` — describes a virtual node's ability to run flows, to have checkpoints, and to continue in-progress flows. </li> <li> `vaultDbOperationalStatus` —  describes a virtual node's ability to perform persistence operations on the virtual node's vault. </li> </ul>These four statuses replace the existing `state` field. The `GET virtualnode` now returns the status of each of the four operational characteristics. Maintenance mode has been updated to disable all four of these new operational statuses. You can set this mode using the existing change virtual node state endpoint: `/api/v1/virtualnode/{virtualNodeShortHash}/state/{newState}`. <br>  **Upgrading a CPI** - A new PUT method has been added to the `virtualnode` resource to upgrade a virtual node's CPI: `/api/v1/virtualnode/{virtualNodeShortHash}/cpi/{target-CPI-file-checksum}`. Before upgrading, the virtual node must be in maintenance mode with no other operations currently in progress. You can check the list of running flows using `GET /api/v1/flow/{virtualNodeShortHash}`. When the virtual node is in maintenance, and when no flows are running (that is, all flows have either "COMPLETED", "FAILED" or "KILLED" status), it is safe to trigger a virtual node upgrade. The target CPI should have the same name, signer summary hash, and MGM group ID as the existing CPI. <br> <br> **Corda CLI** - Any worker and CLI long options that used camel case now use kebab case. This may require that existing scripts are updated. |
```

Comment:
```suggestion
| **Network Operation**                        | **Member Registration Approval** - A network operator can now configure membership groups so that the operator is required to manually approve (or decline) member registration requests. For more information, see [Member Registration Approval]({{< relref "../operating/registration-approval.md" >}}). <br> <br> **Operational Statuses** - Four new operational statuses have been added for virtual nodes to provide fine-grained control over their flows and what components they can interact with: <ul><li>`flowOperationalStatus` — describes a virtual node's ability to start new flows. </li> <li>`flowP2pOperationalStatus` — describes a virtual node's ability to communicate with peers.</li> <li>`flowStartOperationalStatus` — describes a virtual node's ability to run flows, to have checkpoints, and to continue in-progress flows. </li> <li> `vaultDbOperationalStatus` —  describes a virtual node's ability to perform persistence operations on the virtual node's vault. </li> </ul>These four statuses replace the existing `state` field. The `GET virtualnode` now returns the status of each of the four operational characteristics. Maintenance mode has been updated to disable all four of these new operational statuses. You can set this mode using the existing change virtual node state endpoint: `/api/v1/virtualnode/{virtualNodeShortHash}/state/{newState}`. <br><br>  **Upgrading a CPI** - A new PUT method has been added to the `virtualnode` resource to upgrade a virtual node's CPI: `/api/v1/virtualnode/{virtualNodeShortHash}/cpi/{target-CPI-file-checksum}`. Before upgrading, the virtual node must be in maintenance mode with no other operations currently in progress. You can check the list of running flows using `GET /api/v1/flow/{virtualNodeShortHash}`. When the virtual node is in maintenance, and when no flows are running (that is, all flows have either "COMPLETED", "FAILED" or "KILLED" status), it is safe to trigger a virtual node upgrade. The target CPI should have the same name, signer summary hash, and MGM group ID as the existing CPI. <br> <br> **Corda CLI** - Any worker and CLI long options that used camel case now use kebab case. This may require that existing scripts are updated. |
```